addon/components/em-text.js

```diff
@@ -10,12 +10,13 @@ Syntax:
  */
 export default FormGroupComponent.extend({
   controlView: Ember.TextArea.extend(ControlMixin, {
+    attributeBindings: ['placeholder', 'required', 'autofocus', 'disabled'],
```

Comment:
Does the test pass without this line? TextArea already defines this bindings, so this should not be needed.